test/integration/job/job_test.go

```diff
@@ -61,7 +61,10 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-const waitInterval = time.Second
+const (
+	waitInterval        = time.Second
+	NoCompletionsUpdate = -1
```

Comment:
put those inside the test; also not need to export the variable.

Reply:
`waitInterval` is shared by many tests and was already a global constant for this reason, so I didn't move it anywhere, but I did move `noCompletionsUpdate` to be inside `TestElasticIndexedJobs` since it is only used there (and made it non-exported).